src/os.rs

```diff
@@ -311,13 +312,17 @@ mod imp {
             unsafe { mem::transmute(v) }
         }
         fn fill_bytes(&mut self, v: &mut [u8]) {
-            let ret = unsafe {
-                CryptGenRandom(self.hcryptprov, v.len() as DWORD,
-                               v.as_mut_ptr())
-            };
-            if ret == 0 {
-                panic!("couldn't generate random bytes: {}",
-                       io::Error::last_os_error());
+            // CryptGenRandom takes a DWORD (u32) for the length so it only supports
+            // writing u32::MAX bytes at a time.
+            for slice in v.chunks_mut(u32::MAX as usize) {
```

Comment:
Could this use `<DWORD>::max_value()`? That should help avoid the import and also avoid hardcoding u32 == DWORD